fhir-server-test/src/test/resources/fhir-server-config.json

```diff
@@ -0,0 +1,233 @@
+{
+  "__comment": "FHIR Server configuration",
```

Comment:
i've been putting test configurations like this one under `fhir-server-webapp/src/test/liberty/config` ...does it make sense to move this one there and give it a name that indicates what its for?

Reply:
This has been fixed as suggested. Removed the fhir-server-config.json added to fhir-server-test  module and reused existing config "fhir-server-webapp/src/test/liberty/config/config/default/fhir-server-config-audit-cicd.json"

Reply:
great.  its not great that the test depends on this other file in this other module (likely brittle) but I think I like that more than duplicating that file here.  we'll find out in the long run :-)